### ISSUE TITLE
Возможность изменить название диалога

### ISSUE THREAD

**Issue by UsmanShinobi** [rayofhope-dev/dialog-editor#15]
Сейчас нет возможности изменить название диалога. Правда, чтобы замена была полностью автоматизированной, придётся синхронизировать программу с файлами character_desc. 
![image](https://user-images.githubusercontent.com/59365467/193017780-c9880c2a-cc14-4087-836a-c141cd8ae219.png)

**Comment by collectioner** [comment]
Работа с `character_desc` на данный момент выходит за рамки этого редактора. Можешь подсказать, что за `character_desc` файлик и что в нем хранится?
Не он ли читается нашим же NPC Editor-ом?

Переименовать сам диалог, думаю, реально.

**Comment by UsmanShinobi** [comment]
Да, именно он читается НПС-эдитором, и как раз там прописываются диалоги. Кстати, как раз из этих файлов тулза может понять, как используется диалог - как start_dialog или как actor_dialog.
